package.json

```diff
@@ -7,11 +7,11 @@
     "build": "pnpm -r build",
     "start": "pnpm --filter @graphprotocol/docs start",
     "export": "pnpm --filter @graphprotocol/docs export",
-    "lint": "eslint . --ext .js,.jsx,.ts,.tsx,.mjs && pnpm prettier:check && pnpm typecheck",
-    "lint:fix": "eslint . --fix --ext .js,.jsx,.ts,.tsx,.mjs && pnpm prettier && pnpm typecheck",
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx,.mjs; pnpm prettier:check; pnpm typecheck",
+    "lint:fix": "eslint . --fix --ext .js,.jsx,.ts,.tsx,.mjs; pnpm prettier; pnpm typecheck",
```

Comment:
@benface we should use `&&` instead, otherwise, it's not fails if some command fails, e.g. `eslint --max-warnings 0`

Reply:
Ah yeah, oops. I guess I just find it annoying that if e.g. Prettier fails, the TS errors are not even shown. Maybe Prettier should be last?

Reply:
yes, I guess the following commands `typecheck` -> `eslint` -> `prettier` will be better